templates/terraform/custom_import/healthcare_consent_store.go.erb

```diff
@@ -0,0 +1,16 @@
+
+  config := meta.(*Config)
+
+  consentStoreId, err := parseHealthcareConsentStoreId(d.Id(), config)
```

Comment:
I don't see this present in the PR anymore, where does it exist?

Reply:
Removed.